privaterelay/apps.py

```diff
@@ -93,3 +94,9 @@ def fxa_verifying_keys(self) -> list[dict[str, Any]]:
             keys: list[dict[str, Any]] = resp.json()["keys"]
             return keys
         return []
+
+    @cached_property
+    def fxa_social_app(self) -> Any:
```

Comment:
note: I couldn't quite figure out how to annotate this with `-> SocialApp` because Django kept complaining that I couldn't load the `SocialApp` model until the app registry was full.

Reply:
How about https://github.com/mozilla/fx-private-relay/pull/4634/commits/06661e9fe9d846482564d2ae8c48a1fd1681c87e ? (I tried `from __future__ import annotations` first, but it didn't work. 😢